darts/dataprocessing/transformers/window_transformer.py

```diff
@@ -88,49 +78,38 @@ def __init__(
             <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.rolling.html>`_
             and `pandas.DataFrame.expanding().apply() documentation
             <https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.expanding.html>`_.
-
         treat_na
             Specifies how to treat missing values that were added by the window transformations
             at the beginning of the resulting TimeSeries. By default, Darts will leave NaNs in the resulting TimeSeries.
             This parameter can be one of the following:
-
```

Comment:
I think removing these blank lines before a list in rst format might have adverse effect once this is rendered in HTML. Could you perhaps check rendering the doc in HTML (`./gradlew buildDocs`) or - if you want to be on the safe side - simply keep these blank lines.